dev-master/zen/templates/pokedex.py

```
@@ -0,0 +1,182 @@
+        if evolution_path is None:
+            evolution_path = repo_root.parent / "templates" / "metadata" / "evolution.yaml"
+
+        self.evolution_path = evolution_path
+        self._evolution_index = self._load_evolution(evolution_path)
+
+    # ------------------------------------------------------------------
+    # Loading and indexing
+    # ------------------------------------------------------------------
+    def _load_evolution(self, path: Path) -> Dict[str, Dict[str, Any]]:
+        if not path.exists():
+            raise TemplateRegistryError(f"Evolution tracker not found at {path}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)